mailer/template.go

```diff
@@ -55,7 +56,7 @@ func (m *TemplateMailer) InviteMail(user *models.User, referrerURL string) error
 
 	redirectParam := ""
 	if len(referrerURL) > 0 {
-		redirectParam = "&redirect_to=" + referrerURL
+		redirectParam = "&redirect_to=" + url.QueryEscape(referrerURL)
```

Comment:
I think this is correct, but maybe we should be encoding it before it hits the template?

will discuss with @kangmingtay what the best solution is here